### PR TITLE
Add basic blacklist functionality

### PR DESCRIPTION
**Pull request by xXxsomebodyoncetoldmexXx**
Example usecase
`./suo5 -t 'https://example.com/suo5.aspx' -l '127.0.0.1:1080' -B 'polling.oastify.com,portswigger.net'`

```
[INFO] 06-17 16:38 starting tunnel at 0.0.0.0:1080

[Tunnel Info]
Target:  http://example.com/suo5.aspx
Proxy:   socks5://0.0.0.0:1080
Mode:    half

[INFO] 06-17 16:38 creating a test connection to the remote target
[INFO] 06-17 16:38 start connection to 127.0.0.1:0
[DBUG] 06-17 16:38 recv dial response from server: length: 7
[INFO] 06-17 16:38 successfully connected to 127.0.0.1:0
[DBUG] 06-17 16:38 recv socks5 reply: 0
[DBUG] 06-17 16:38 local conn closed, 127.0.0.1:0
[INFO] 06-17 16:38 congratulations! everything works fine
[DBUG] 06-17 16:38 remote readwriter closed, 127.0.0.1:0
[INFO] 06-17 16:38 connection closed, 127.0.0.1:0
[SUCC] 06-17 16:38 drop connecction to polling.oastify.com:443
[SUCC] 06-17 16:38 drop connecction to polling.oastify.com:443
[SUCC] 06-17 16:39 drop connecction to portswigger.net:443
```

### CODE REVIEW

File: ctrl/handler.go [hunk 1]
```diff
@@ -52,6 +56,23 @@ func (m *socks5Handler) Handle(conn net.Conn) error {
...
+		Title:     "[SUCC]",
+		ColorCode: 32,
+	}
+
+	sucLog := log.New()
+	sucLog.SetLevel("success")
+	sucLog.SetTimeFormat("01-02 15:04")
+
+	if slices.Contains(BlackUrl, strings.Split(req.Addr.String(), ":")[0]) {
+		sucLog.Logf(SuccessLevel, "drop connecction to %s", req.Addr.String())
```
Comment (zema1) [inline ctrl/handler.go:72]: 这里 Info 或者 Warn 我感觉都可以接受，没必要重新定义一个 log level

**Comment by zema1** [comment]
抱歉这些天一直没有评论，这个功能挺好的，可以在再改一下我合了

File: ctrl/handler.go (hunk 1 above)
Comment (xXxsomebodyoncetoldmexXx) [reply ctrl/handler.go:72]: yeah I just want the block output to have different color, will change into Info log level

**Comment by xXxsomebodyoncetoldmexXx** [comment]
Hi, i change a bit so that it match the project coding style, now the syntax is like this
`./suo5 -d --nh -t 'http://192.168.30.28/suo5.aspx' -l '0.0.0.0:1080' -E "google.com" -E "stackoverflow" -E "amazon.com"`

image

**Pull request merged**